### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by RomanPerepelkin**
- [DEMO LINK](https://RomanPerepelkin.github.io/layout_stars/)
- [TEST REPORT LINK](https://RomanPerepelkin.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -8,6 +8,54 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <!-- <h1>Stars</h1> -->
```
Comment (nazarmatsevych) [inline src/index.html:11]: remove comments

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]
No need to add margin-right to the last elements in a row
image

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars-star:last-child {
+  margin-right: 0;
+}
+
+.stars--5 .stars-star,
+.stars--1 .stars-star:first-child,
```
Comment (OleziO) [inline src/style.css:24]: Replace with `-n + 1` and `-n  + 5` to make all code in the same style.

**Review by OleziO — CHANGES_REQUESTED** [review]

**Review by nazarmatsevych — APPROVED** [review]
👍

**Pull request closed without merging**